backend/fms_core/template_importer/row_handlers/sample_qc/sample.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime
+
+from fms_core.template_importer.row_handlers._generic import GenericRowHandler
+
+from fms_core.services.sample import get_sample_from_container, update_sample
+from fms_core.services.process_measurement import create_process_measurement
+from fms_core.services.property_value import create_process_measurement_properties
+from fms_core.services.derived_sample import update_qc_flags
+from fms_core.models import InstrumentType
+
+INSTRUMENT_PROPERTIES = ['Electrophoresis Instrument', 'Quantitation Instrument']
+TYPES_BY_PLATFORM = {
+    'PicoGreen': 'Quality Control: Quantitation',
+    'Qubit': 'Quality Control: Quantitation',
+    'Agarose Gel': 'Quality Control: Electrophoresis',
+    'TapeStation': 'Quality Control: Electrophoresis',
+}
+
+
+class SampleQCRowHandler(GenericRowHandler):
+    def __init__(self):
+        super().__init__()
+
+    def process_row_inner(self, sample, sample_information, process_measurement, process_properties):
+        sample_obj, self.errors['sample'], self.warnings['sample'] = get_sample_from_container(
+            barcode=sample['container']['barcode'],
+            coordinates=sample['coordinates'],
+        )
+
+        if sample_obj:
+            # Update sample with sample_information
+            new_volume = None
+            # TODO: Maybe validate initial volume?
+            if all([sample_information['initial_volume'], sample_information['measured_volume'],
+                    process_measurement['volume_used']]):
+                delta_volume = sample_information['initial_volume'] - sample_information['measured_volume']
```

Comment:
I would reverse the 2 values... i would expect if the measured_volume is superior to the initial _volume to have a positive Delta_volume ...